src/core/utils/nfts.ts

```diff
@@ -135,6 +137,12 @@ export function getNetworkFromSimpleHashChain(
       return ChainName.forma;
     case SimpleHashChain.Godwoken:
       return ChainName.godwoken;
+    case SimpleHashChain.Gnosis:
```

Comment:
I'm not sure why gnosis was on mainnet before, was this intentional?

Reply:
@walmat This was for POAPs because they migrated to Gnosis chain

Reply:
reverted